downloads/swan-lake-release-notes/swan-lake-security-fixes-template/RELEASE_NOTE.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: ballerina-left-nav-release-notes
+title: Swan Lake Update 8 (<RELEASE_VERSION>)
+permalink: /downloads/swan-lake-release-notes/<RELEASE_VERSION>/
+active: 2201.8.2
+redirect_from: 
+    - /downloads/swan-lake-release-notes/<RELEASE_VERSION>
+    - /downloads/swan-lake-release-notes/<RELEASE_VERSION>-swan-lake/
+    - /downloads/swan-lake-release-notes/<RELEASE_VERSION>-swan-lake
+    - /downloads/swan-lake-release-notes/
+    - /downloads/swan-lake-release-notes
+---
+
+## Overview of Ballerina Swan Lake Update 8 (<RELEASE_VERSION>)
+
+<!-- COMMENT OUT ONLY THE APPLICABLE ONE FROM THE BELOW -->
+
+<!-- <em>Swan Lake Update 8 (<RELEASE_VERSION>) is the <RELEASE_SEQUENCE_NUMBER> patch release of Swan Lake Update 8 (2201.8.0) and it includes a security improvement.</em>
+
+<em>Swan Lake Update 8 (<RELEASE_VERSION>) is the <RELEASE_SEQUENCE_NUMBER> patch release of Swan Lake Update 8 (2201.8.0) and it includes a few security improvements.</em> -->
+
+## Update Ballerina
+
+Update your current Ballerina installation directly to <RELEASE_VERSION> using the [Ballerina Update Tool](/learn/update-tool/) as follows.
+
+1. Run `bal update` to get the latest version of the Update Tool.
+2. Run `bal dist update` to update to this latest distribution.
```

Comment:
Lets remove `bal update` and include `bal dist update` only

Reply:
Updated.